deploy/docs/Non_Helm_Installation.md

```diff
@@ -5,17 +5,16 @@ This document has instructions for setting up collection with Fluentd, FluentBit
 <!-- TOC -->
 
 - [Before you start](#before-you-start) 
-- [Create Sumo Fields, a collector, and deploy Fluentd](#create-sumo-fields-a-collector-and-deploy-fluentd) 
-  - [Automatic Source Creation and Setup Script](#automatic-source-creation-and-setup-script) 
-    - [Parameters](#parameters) 
-    - [Environment variables](#environment-variables) 
+- [Create Sumo Fields and a collector](#create-sumo-fields-a-collector) 
```

Comment:
We'll need to update line 46 to reflect the new header as well

Reply:
good catch